src/main/java/duke/task/Event.java

```diff
@@ -6,24 +6,43 @@ public class Event extends Task {
     LocalDateTime start;
     LocalDateTime end;
 
+    /**
+     * Constructor for Event.
```

Comment:
Perhaps leave a line between method description and param fields in the documentation for better readability?